site/src/components/Resources/Resources.tsx

```diff
@@ -103,40 +100,43 @@ export const Resources: FC<React.PropsWithChildren<ResourcesProps>> = ({
                       )}
 
                       <TableCell className={styles.agentColumn}>
-                        {agent.name}
-                        <div className={styles.agentInfo}>
-                          <span className={styles.operatingSystem}>{agent.operating_system}</span>
-                          {WorkspaceStateEnum[workspaceStatus] !==
-                            WorkspaceStateEnum["stopped"] && (
-                            <span style={{ color: agentStatus.color }} className={styles.status}>
+                      <TableCellDataPrimary highlight>{agent.name}</TableCellDataPrimary>
+                      <div className={styles.data}>
+                        <div className={styles.dataRow}>
+                          <strong>status: </strong>
+                          <span style={{ color: agentStatus.color }} className={styles.status}>
                               {agentStatus.status}
                             </span>
-                          )}
                         </div>
+                        <div className={styles.dataRow}>
+                          <strong>os: </strong>
+                          {agent.operating_system}
+                        </div>
+                      </div>
                       </TableCell>
                       <TableCell>
-                        <>
-                          {canUpdateWorkspace && agent.status === "connected" && (
-                            <div className={styles.accessLinks}>
-                              <SSHButton workspaceName={workspace.name} agentName={agent.name} />
-                              <TerminalLink
-                                workspaceName={workspace.name}
-                                agentName={agent.name}
-                                userName={workspace.owner_name}
-                              />
-                              {agent.apps.map((app) => (
-                                <AppLink
-                                  key={app.name}
-                                  appIcon={app.icon}
-                                  appName={app.name}
-                                  userName={workspace.owner_name}
-                                  workspaceName={workspace.name}
-                                  agentName={agent.name}
-                                />
-                              ))}
-                            </div>
-                          )}
-                        </>
+                          <div className={styles.accessLinks}>
```

Comment:
I did this to try to get this column to stay the same width even if this condition isn't true. I'm not sure if it really worked.